src/components/HelpDrawer/HelpContent/SpectralProfilerHelpContent.tsx

```diff
@@ -9,78 +9,76 @@ export const SPECTRAL_PROFILER_HELP_CONTENT = (
+            In short, if multiple spectra are plotted, only one option (<b>Image</b>, <b>Region</b>, <b>Statistic</b> or <b>Polarization</b>) can be varied at a time. All other options are fixed to a single value.
         </p>
+        <p>The cursor information of each profile is displayed in the bottom-left corner. The cursor information field can be resized.</p>
 
         <h3 id="images">Image dropdown menu</h3>
         <p>
-            The image dropdown menu defaults to &quot;Active&quot; image which means the current image in the image viewer if it is in single-panel mode. If it is in multi-panel mode, the active image is highlighted with a red box. You may
-            use the animator widget or the image list widget to change the active image.
+            The <b>Image</b> dropdown menu defaults to &quot;Active&quot; image which means the current image in the Image Viewer if it is in single-panel mode. If it is in multi-panel mode, the active image is highlighted with a red box.
+            You may use the Animator Widget or the Image List Widget to change the active image.
         </p>
         <p>
-            When the <code>Image</code> checkbox is selected and if there are spatially and spectrally matched images (apply matching via the image list widget), the dropdown menu will display an image list with information about the
-            matching state. You can select one of the images to view its spectral profile. If the selected image is matched to other images, spectra from those images will be displayed in the spectral profiler widget too, allowing a direct
+            When the <b>Image</b> checkbox is selected and if there are spatially and spectrally matched images (apply matching via the Image List Widget), the dropdown menu will display an image list with information about the matching
+            state. You can select one of the images to view its spectral profile. If the selected image is matched to other images, spectra from those images will be displayed in the Spectral Profiler Widget too, allowing a direct
             comparison of spectra from different image cubes.
         </p>
         <h3 id="regions">Region dropdown menu</h3>
         <p>
-            The region dropdown menu defaults to &quot;Active&quot; region which means the selected region in the image viewer. You can select an active region by clicking one on the image viewer, or by clicking a region entry in the region
-            list widget. The spectral profile plot of the selected active region will be updated accordingly. If no region is selected, the region defaults to cursor.
+            The <b>Region</b> dropdown menu defaults to &quot;Active&quot; region which means the selected region in the Image Viewer. You can select an active region by clicking one in the Image Viewer, or by clicking a region entry in the
```

Comment:
`The <b>Region</b> dropdown menu defaults to "Active", which means the region currently selected in the Image Viewer.`

Reply:
also applied the same elsewhere